qa/tasks/persistent_write_log_cache_thrash.py

```diff
@@ -0,0 +1,87 @@
+"""
+persistent write log cache thrash task
+"""
+import contextlib
+import logging
+
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from teuthology.orchestra import run
+
+DEFAULT_KILL_DAEMON_TIME = 2
+DEFAULT_DEAD_TIME = 60
+DEFAULT_LIVE_TIME = 10
+
+log = logging.getLogger(__name__)
+
+@contextlib.contextmanager
+def thrashes_rbd_bench_on_persistent_cache(ctx, config):
+    """
+    thrashes rbd bench on persistent write log cache.
+    It can test recovery feature of persistent write log cache.
+    """
+    log.info("thrashes rbd bench on persistent write log cache")
+
+    client, client_config = list(config.items())[0]
+    (remote,) = ctx.cluster.only(client).remotes.keys()
+    client_config = client_config if client_config is not None else dict()
+    kill_daemon_time = client_config.get('kill_daemon_time', DEFAULT_KILL_DAEMON_TIME)
+    dead_time = client_config.get('dead_time', DEFAULT_DEAD_TIME)
+    live_time = client_config.get('live_time', DEFAULT_LIVE_TIME)
+    rbd_status_command = '"$(rbd status testimage | grep "Image cache state")"'
```

Comment:
Is following single quotes by double quotes actually needed?

Reply:
Yes, needed. single quotes make the sentence as a python string. double quotes is the parameter of `test -n`. This is a format for shell pipes. And the teuthology output:
`2022-01-17T15:36:09.923 DEBUG:teuthology.orchestra.run.plana303:> test -n "$(rbd status testimage | grep "Image cache state")"`